src/components/DefinitionList/DefinitionList.scss

```diff
@@ -73,6 +93,7 @@ $block: '.#{variables.$ns}definition-list';
     &__copy-container {
         position: relative;
         display: inline-flex;
+        align-items: center;
```

Comment:
It's unaligned for single line value:
<img width="358" alt="" src="https://github.com/gravity-ui/components/assets/3368839/59f73f0c-36bb-4b6c-b105-b4205670c18a">

Reply:
Somehow storybook wasn't updated. Please take a look now!